### PR TITLE
Mirror IE + Chrome -> Edge for html/

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR is a part of #5214, in mirroring Chromium data over to Edge as per the new Edge version.  This performs four tasks: setting the existing `true` values to `≤18`, mirroring the Chromium data to any entires marked as `null` whilst setting the version to `≤79`, mirroring the Chromium data to entries marked as `false`, and setting anything that's `true` in IE to `12` in Edge.  The majority of this data has been generated by the mirroring script, however due to the nature of the migration, there's also a good portion modified manually (I do plan on upgrading the mirroring script to work better for this).

### CODE REVIEW

File: html/elements/keygen.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
               "version_removed": "57"
             },
             "edge": {
-              "version_added": true
+              "version_added": "≤18"
```
Comment (Elchi3) [inline html/elements/keygen.json]: I think we need to add` "version_removed": "79"` here.

File: html/elements/meta.json [hunk 2]
```diff
@@ -536,7 +536,8 @@
                   "notes": "Until Chrome 46, <code>content</code> values weren't constrained to the values listed in the spec."
                 },
                 "edge": {
-                  "version_added": false
+                  "version_added": "79",
+                  "notes": "Until Edge 46, <code>content</code> values weren't constrained to the values listed in the spec."
```
Comment (Elchi3) [inline html/elements/meta.json]: This note makes no sense. There is no Edge 46. We should remove it.

File: html/global_attributes.json [hunk 3]
```diff
@@ -130,9 +130,16 @@
...
-              "version_added": null
-            },
+            "edge": [
+              {
+                "version_added": "≤79",
+                "notes": "Support added for <code>autocomplete</code> on the <code>&lt;textarea&gt;</code> and <code>&lt;select&gt;</code> elements."
+              },
+              {
+                "version_added": "≤79",
+                "notes": "Originally only supported on the <code>&lt;input&gt;</code> element."
```
Comment (Elchi3) [inline html/global_attributes.json]: I think these notes and therefore the range doesn't apply to Edge. Can be just version_added: 79. Do you think so, too?

File: html/global_attributes.json [hunk 4]
```diff
@@ -574,7 +581,15 @@
...
-              "version_added": false
+              "version_added": "79",
+              "flags": [
+                {
+                  "type": "runtime_flag",
+                  "name": "--enable-blink-features",
+                  "value_to_set": "ContextMenu"
+                }
+              ],
+              "notes": "This was removed from the <i>Enable Experimental Web Platform Features</i> due to a <a href='https://crbug.com/412945'>Web compatibility issue</a>. In June 2017, it was removed entirely from the browsers. This is documented in <a href='https://crbug.com/87553'>Chromium bug 87553</a>."
```
Comment (Elchi3) [inline html/global_attributes.json]: This note makes no sense for Edge, I think.

File: html/elements/keygen.json (hunk 1 above)
Comment (vinyldarkscratch) [reply html/elements/keygen.json]: Oops!  I must have missed this scenario where Chrome has a `version_removed` value and there was support within EdgeHTML.  I'll try to programmatically fix this case so all the mirroring PRs will be fixed!

**Comment by Elchi3** [comment]
This looks good to me now! Can you comment on your strategy for this:

> I don't know of a case in html/, but how do you deal with features that are supported in say Edge 17, but aren't supported in Chrome. I don't see a range (added in 17 and removed in 79). Maybe this case doesn't happen here, or isn't it taken into account?

I want to make sure we're not missing things here.

**Comment by vinyldarkscratch** [comment]
At first I was planning on following up in some new PRs to get these ones merged right away, but I’m now realizing it’d be better to tackle it in one go.  I’m going to get another data mirror going to set a version_removed any truthy values in Edge that are falsy in Chrome.  Furthermore, I also plan on performing a mirroring that copies data from Chrome 80-82 over to Edge 80-82, since Chrome/Edge 80 are now released.

So, this PR’s ready to be merged if you’d like to get the Edge Chromium data going, or if you prefer to hold off I can improve it some more!

**Comment by vinyldarkscratch** [comment]
Updates are now complete!

**Comment by Elchi3** [comment]
Well done! The version_removed stuff looks good to me now. :tada: 

One more question: If a feature is supported in old Edge, but not supported in Chrome, how does the script handle it? It seems this case doesn't happen here?

**Comment by vinyldarkscratch** [comment]
Huh, I had thought that I caught those scenarios -- I had programmed the script to set `"version_removed": "79"` in Edge for anything with `"version_added": false` in Chrome (and I had just gone through the entries manually for anything that has a `version_removed` in Chrome).  Was there a specific line you saw that didn't seem to have this?  (I'll be happy to fix it right away!)

**Comment by Elchi3** [comment]
> I had programmed the script to set "version_removed": "79" in Edge for anything with "version_added": false in Chrome 

Thanks! This is what I wanted to know. I don't see this case in this PR that's why I'm asking if you've thought about it. (I only see "version_removed": "79" when Chrome has "version_removed" but not when Chrome has "version_added": false.)

**Comment by Elchi3** [comment]
Founds these cases now:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/basefont#Browser_compatibility
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu

These things were supported in Edge, but are now gone in Chromium based Edge. I don't see that reflected in this PR.

**Comment by vinyldarkscratch** [comment]
Oh huh, I guess it didn't go through after all.  I'll get right on that!

**Comment by vinyldarkscratch** [comment]
Alright, all done!  All the mirroring PRs now have the `version_removed` fixed.  :wink:

**Pull request merged**